openapi_python_client/templates/model.pyi

```diff
@@ -25,37 +26,37 @@ class {{ model.reference.class_name }}:
     """ {{ model.description }} """
     {% for property in model.required_properties + model.optional_properties %}
     {% if property.default is none and property.required %}
-    {{ property.to_string() }}
+    _{{ property.to_string() }}
     {% endif %}
     {% endfor %}
     {% for property in model.required_properties + model.optional_properties %}
     {% if property.default is not none or not property.required %}
-    {{ property.to_string() }}
+    _{{ property.to_string() }}
     {% endif %}
     {% endfor %}
     {% if model.additional_properties %}
-    additional_properties: Dict[str, {{ additional_property_type }}] = attr.ib(init=False, factory=dict)
+    _additional_properties: Dict[str, {{ additional_property_type }}] = attr.ib(init=False, factory=dict)
```

Comment:
Not for any technical reasons, more for consistency in having backing members be `_`-prefixed.  My assumption is that these are normally accessed via the dictionary accessors, but if not, it probably makes sense to leave them undecorated.

Reply:
`additional_properties` is in a weird middle ground between implementation detail that you don't need to know about and something you should care about.

But it's infrequent/never that we directly access `additional_properties` - we use the passthroughs, so I think this is good

Reply:
I agree with that logic, but I just realized that this is going to make upgrading harder. I just did a quick search of `.additional_properties` on GitHub and found a bunch of usages, such as:
```
https://github.com/benchling/integration-corteva-assays/blob/b9e6851ca764bb1c5c66306b14c773e1657c5292/modules/modules/plates/parsers.py#L21
```

Since people tend to find upgrading the SDK very painstaking already, I think the team would prefer if we minimize breaking changes until necessary. If we want to make this change, maybe we can lump it into the GA release (which will have a ton of other breaking changes anyway)?

Reply:
Unprefixed.